exercises/palindrome-products/tests/palindrome-products.rs

```diff
@@ -0,0 +1,58 @@
+extern crate palindrome_products;
+use palindrome_products::*;
+
+#[test]
+#[ignore]

```

Comment:
The first test in the tests file should be the one not ignored. There are at least two ways to accomplish this: 

- move the `single_digits` test to the top of the file.
- ignore `single_digits` and unignore `empty_result_for_smallest_palindrome`.

I do not have a preference between these options.